admin/templates/configuration/tmp/configure/sql/qserv-worker.sql

```diff
@@ -1,13 +1,42 @@
 CREATE USER IF NOT EXISTS '{{MYSQLD_USER_QSERV}}'@'localhost';
 
 -- Used by xrootd Qserv plugin:
--- to publish LSST databases
+-- to publish LSST databases, tables and chunks
+
 DROP DATABASE IF EXISTS qservw_worker;
 CREATE DATABASE qservw_worker;
+
 GRANT SELECT ON qservw_worker.* TO '{{MYSQLD_USER_QSERV}}'@'localhost';
+
 CREATE TABLE qservw_worker.Dbs (
-  `db` char(200) NOT NULL
-);
+
+  `db` CHAR(200) NOT NULL,
+
+  PRIMARY KEY (`db`)
+
+) ENGINE=InnoDB;
+
+CREATE TABLE qservw_worker.Chunks (
+
+  `db`    CHAR(200)    NOT NULL,
+  `table` CHAR(200)    NOT NULL,
+  `chunk` INT UNSIGNED NOT NULL,
+
+  UNIQUE KEY(`db`,`table`,`chunk`)
+
+) ENGINE=InnoDB;
```

Comment:
No, the table names are not needed by the replication system. There was a different reason to have that extra column. In the present implementation of wpublish (class ChunkInventory) chunks are registered in the following dictionary:
[database][chunk](table1,table2,table3,...)
I decided to leave the tables there since I'm not sure who else may rely on them.
Though, you're right - I should probably have to get rid of those 'tables' altogether (in the schema of my new table and in the code).
I'd be happy to do so. Would you recommend this?

Reply:
I got rid of 'table' everywhere (schema, code).